rubocop.yml.tt

```diff
@@ -100,3 +101,7 @@ Style/StringLiterals:
 
 Style/TrivialAccessors:
   AllowPredicates: true
+
+RSpec/MultipleExpectations:
+  Max: 2
```

Comment:
I think that two expectations in one test is reasonable, especially in system tests.

Reply:
Yes. The only time this can be pushed over is if there are `expect(...). to receive`